hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/streams/assertions/EventualRecordStreamAssertion.java

```diff
@@ -101,6 +130,12 @@ public void assertHasPassed() {
         }
     }
 
+    public void unsubscribe() {
+        if (unsubscribe != null) {
```

Comment:
Or a do-nothing lambda could have been assigned to `unsubscribe` in the first place so it could always be called ...